hack/test-update-storage-objects.sh

```diff
@@ -107,17 +107,16 @@ echo "${ETCD_VERSION}/${STORAGE_BACKEND_ETCD2}" > "${ETCD_DIR}/version.txt"
 
 ### BEGIN TEST DEFINITION CUSTOMIZATION ###
 
-# After removing HPA in v1beta1 there is basically no resource which has
-# two versions. In the future we may want to extend it.
+# Set "namespace" field to "cluster-scoped" if needed.
 # source_file,resource,namespace,name,old_version,new_version
 tests=(
-test/fixtures/doc-yaml/user-guide/pod.yaml,pods,default,nginx,v1,v1
+examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,cluster-scoped,slow,v1beta1,v1
 )
 
-KUBE_OLD_API_VERSION="v1,extensions/v1beta1"
```

Comment:
I know this test passed previously, but I have a feeling it was coincidental given that the previous value of this env var is in the wrong format.